dask/dataframe/io/tests/test_io.py

```diff
@@ -496,15 +496,29 @@ def test_to_bag():
     assert ddf.x.to_bag().compute() == list(a.x)
 
 
-def test_to_records():
+@pytest.mark.parametrize('lengths', [
+    None,
+    [2, 3],
+    True,
+])
+def test_to_records(lengths):
     pytest.importorskip('dask.array')
     from dask.array.utils import assert_eq
     df = pd.DataFrame({'x': ['a', 'b', 'c', 'd'],
                        'y': [2, 3, 4, 5]},
                       index=pd.Index([1., 2., 3., 4.], name='ind'))
     ddf = dd.from_pandas(df, 2)
 
-    assert_eq(df.to_records(), ddf.to_records())
+    assert_eq(df.to_records(), ddf.to_records(lengths=lengths))
+
+    # assert isinstance(result, da.Array)
```

Comment:
Was this failing if not commented out?

Reply:
Sorry you are correct, this does not fail when not commented.